src/components/ProjectFilterOptionsDialog/ProjectFilterOptionsDialog.tsx

```diff
@@ -0,0 +1,93 @@
+import { Box, makeStyles } from '@material-ui/core';
+import * as React from 'react';
+import { ProjectStatus, Sensitivity } from '../../api';
+import { displayStatus } from '../../api/displayStatus';
+import { DialogForm, DialogFormProps } from '../Dialog/DialogForm';
+import { RadioField, RadioOption, SubmitError } from '../form';
+import { SwitchField } from '../form/SwitchField';
+
+const useStyles = makeStyles(({ spacing, typography }) => ({
+  checkboxGrid: {
+    display: 'flex',
+    flexFlow: 'row wrap',
+    alignItems: 'center',
+    paddingTop: spacing(2),
+  },
+  fieldLabel: {
+    fontWeight: typography.weight.bold,
+    marginBottom: spacing(2),
+  },
+}));
+
+interface FormValues {
+  status: ProjectStatus | 'ShowAll';
+  sensitivity: Sensitivity | 'ShowAll';
+  clusters: boolean;
+}
+
+export type ProjectFilterOptionsDialogProps = DialogFormProps<FormValues>;
+
+export const ProjectFilterOptionsDialog = (
+  props: ProjectFilterOptionsDialogProps
+) => {
+  const classes = useStyles();
+
+  console.log('props', props);
+
+  const initialValues: FormValues = props.initialValues
+    ? props.initialValues
+    : {
+        status: 'ShowAll',
+        sensitivity: 'ShowAll',
+        clusters: false,
+      };
```

Comment:
Final Form has `initialValues`, but also `defaultValue` on each field. For this we actually want default values - the values used if there wasn't one previously. Where `initialValues` is actually the previous values when the form/dialog was opened.